android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -213,18 +213,23 @@ class ScreenStackFragment : ScreenFragment {
             // and also this is not necessary when going back since the lifecycle methods
             // are correctly dispatched then.
             // We also add fakeAnimation to the set of animations, which sends the progress of animation
-            val fakeAnimation = ScreensAnimation(mFragment)
-            fakeAnimation.duration = animation.duration
+            val fakeAnimation = ScreensAnimation(mFragment).apply { duration = animation.duration }
+
             if (animation is AnimationSet && !mFragment.isRemoving) {
-                animation.addAnimation(fakeAnimation)
-                animation.setAnimationListener(mAnimationListener)
-                super.startAnimation(animation)
+                animation.apply {
+                    addAnimation(fakeAnimation)
+                    setAnimationListener(mAnimationListener)
+                }.also {
```

Comment:
Why is `also` used here?

Reply:
Separation of object configuration from introducing side effects. [Docs specify it this way](https://kotlinlang.org/docs/scope-functions.html#function-selection), however it is not strict I believe.